### PR TITLE
SDIT-2353: ✨ Add in gotenberg and start on download

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: helm_deploy/hmpps-historical-prisoner/values.yaml [hunk 1]
```diff
@@ -71,3 +71,50 @@ generic-service:
 
 generic-prometheus-alerts:
   targetApplication: hmpps-historical-prisoner
+
+gotenberg:
```
Comment (petergphillips) [inline helm_deploy/hmpps-historical-prisoner/values.yaml:75]: copied from manage adjudications

File: server/@types/express/index.d.ts [hunk 2]
```diff
@@ -51,6 +52,18 @@ export declare global {
     interface Locals {
       user: HmppsUser
     }
+
+    interface Response {
```
Comment (petergphillips) [inline server/@types/express/index.d.ts:56]: copied from manage adjudications

File: server/data/gotenbergClient.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import superagent from 'superagent'
+
+export type PdfMargins = {
+  marginTop?: string
+  marginBottom?: string
+  marginLeft?: string
+  marginRight?: string
+}
+
+export default class GotenbergClient {
```
Comment (petergphillips) [inline server/data/gotenbergClient.ts:10]: copied from manage adjudications

File: server/utils/pdfRenderer.ts [hunk 4]
```diff
@@ -0,0 +1,45 @@
+import { Request, Response, NextFunction } from 'express'
+import GotenbergClient, { PdfMargins } from '../data/gotenbergClient'
+
+export type PdfPageData = { adjudicationsUrl: string } & Record<string, unknown>
+export type PdfHeaderData = Record<string, unknown>
+export type PdfFooterData = Record<string, unknown>
+
+export function pdfRenderer(client: GotenbergClient) {
```
Comment (petergphillips) [inline server/utils/pdfRenderer.ts:9]: copied from manage adjudications

File: server/views/pages/pdfFooter.njk [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+            display: flex;
+            justify-content: end;
+            align-items: center;
+            padding-left: 30px;
+            padding-right: 30px;
+        }
+    </style>
+</head>
+<body>
+    <div class='adjudication-pdf-footer'>
```
Comment (petergphillips) [inline server/views/pages/pdfFooter.njk:24]: blatant copy - needs some tlc

File: server/views/pages/pdfHeader.njk [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+            font-weight: bold;
+        }
+        .adjudication-pdf-header-normal {
+            font-weight: normal;
+        }
+    </style>
+</head>
+<body>
+    <div class='adjudication-pdf-header'>
+        <h1>Conduct report</h1>
```
Comment (petergphillips) [inline server/views/pages/pdfHeader.njk:34]: blatant copy for now

**Review by karenmillermoj — APPROVED** [review]
lgtm

**Pull request merged**